xcp_d/workflow/bold.py

```diff
@@ -409,15 +409,20 @@ def init_boldpostprocess_wf(
         Function(
             input_names=["bold_file", "mni_to_t1w", "t1w_to_native"],
             output_names=["transform_list"],
-            function=get_transformfile,
+            function=get_std2bold_xforms,
         ),
         name="get_std2native_transform",
     )
     get_native2space_transforms = pe.Node(
         Function(
             input_names=["bold_file", "mni_to_t1w", "t1w_to_native"],
-            output_names=["bold2MNI_trans", "bold2T1w_trans"],
-            function=get_transformfilex,
+            output_names=[
+                "bold_to_std_xforms",
```

Comment:
do you know what itf means?

Reply:
I'm using it here as "invert transform flags" because the Nipype interface's parameter is `invert_transform_flags`.

Reply:
ohhh that makes sense. Might as well spell it out, I was thinking it was a typo for "itk"

Reply:
I've changed it to `invert`. Thanks.